src/bstring.ml

```diff
@@ -1,13 +1,25 @@
 
```

Comment:
We should, but the way things are moving I think we need to prioritize organization and settling on an interface first.